README.md

```diff
@@ -855,6 +856,18 @@ curl -X GET http://cloud.foobar.com/.well-known/apple-app-site-association
 }
 ```
 
+## BI web auth
+
+For some banks integration, BI need something similar to universal link because
```

Comment:
"some" ? which ? Do you have any example ? Could you add a link to Budget-insight's doc explaining more in details this mechanism ?